katello/katello/backup.rb

```diff
@@ -299,7 +299,14 @@ def generate_metadata
     end
 
     def create_pulp_data_tar
-      run_cmd("tar --selinux --create --file=#{File.join(@dir, 'pulp_data.tar')} --exclude=var/lib/pulp/katello-export --listed-incremental=#{File.join(@dir, '.pulp.snar')} --transform 's,^,var/lib/pulp/,S' -S *")
+      @archive_size = @options[:split_pulp_tar] || "0"
+      @split_pulp_tar_script = @options[:split_pulp_tar_script] || '/usr/share/katello/katello-backup-rotate-tar.sh'
```

Comment:
having this hardcoded to `/usr/share/katello` does make it hard to test things out in development. You could do something like we have [here](https://git.io/vNFwU) maybe in a method that is added to helper.rb

Reply:
Can do